test/functional/p2p_blockfilters.py

```diff
@@ -94,7 +94,7 @@ def run_test(self):
         self.nodes[0].syncwithvalidationinterfacequeue()
 
         main_block_hash = self.nodes[0].getblockhash(1000)
-        assert main_block_hash != stale_block_hash, "node 0 chain did not reorganize"
+        assert_not_equal(main_block_hash, stale_block_hash, "node 0 chain did not reorganize")
```

Comment:
that is strange I would think that the 3rd arg would make the tests fail in this case

Reply:
I've updated the function to not accept the `args` param since it did not seem to be needed and instead allowed an optional 3rd param as the message

updated in [4488120](https://github.com/bitcoin/bitcoin/pull/29500/commits/4488120ab7db775ce0ae8a73f910dddca1dee123)